Tasks/ECRPushImage/TaskParameters.ts

```diff
@@ -19,6 +19,7 @@ export interface TaskParameters {
     repositoryName: string
     pushTag: string
     autoCreateRepository: boolean
+    enforceDockerNamingConventions: boolean
```

Comment:
This should be "force" vs "enforce" since we are mutating the input string

Reply:
Good catch. Should be updated now.